chemicaltoolbox/chemfp/mol2fps.xml

```diff
@@ -1,21 +1,20 @@
-<tool id="ctb_chemfp_mol2fps" name="Molecules to Fingerprints" version="0.3.0">
-    <description>with different fingerprint types</description>
+<tool id="ctb_chemfp_mol2fps" name="Molecule to fingerprint" version="1.5">
+    <description>conversion to 10 different fingerprint formats</description>
```

Comment:
I'm not sure we should mention the number 10 here. This will maybe change in the future? Any reason you included it?